### PR TITLE
Update the main description of the chef server

### PR DESCRIPTION
**Pull request by tas50**
API service not front end server. It provides the API. That's a better description of what it does
Dont' talk about erchef. That's a term that existed when we had a ruby chef server and an erlang chef server and it needs to die.

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Comment by kagarmoe** [comment]
(ignore the 2 lint tests for now, they're still WIP)

File: _vendor/github.com/chef/chef-server/docs-chef-io/content/server/_index.md [hunk 1]
```diff
@@ -17,7 +17,8 @@ aliases = ["/server_overview.html", "/server_components.html", "/server_overview
 
 {{% chef_server %}}
 
-{{% chef_server_component_erchef_background %}}
+The Chef Infra Server API is written in [Erlang](http://www.erlang.org/), which is an excellent language for critical enterprise concerns like concurrency, fault-tolerance, and distributed
+environments. The Chef Infra Server can scale to the size of any enterprise.
```
Comment (kagarmoe) [inline _vendor/github.com/chef/chef-server/docs-chef-io/content/server/_index.md]: This change needs to happen over in Chef Infra Server.

**Comment by kagarmoe** [comment]
See https://github.com/chef/chef-server/pull/2874

File: _vendor/github.com/chef/chef-server/docs-chef-io/content/server/_index.md (hunk 1 above)
Comment (tas50) [reply _vendor/github.com/chef/chef-server/docs-chef-io/content/server/_index.md]: I'm a big fan of moving the product stuff into the product so this works for me

**Review by kagarmoe — APPROVED** [review]

**Pull request merged**